openlane/flows/flow.py

```diff
@@ -454,10 +457,18 @@ def start(
 
         :returns: ``(success, state_list)``
         """
-        if last_run:
-            if tag is not None:
-                raise FlowException("tag and last_run cannot be used simultaneously.")
+        if last_run and tag is not None:
+            raise FlowException("tag and last_run cannot be used simultaneously.")
 
+        if run_dir is not None and tag is not None:
```

Comment:
Run dir should not be part of the API either I think. It should be _force_run_dir